rust/deny-check/start.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+if [ -z "$GITHUB_WORKSPACE" ]; then
+    echo "GITHUB_WORKSPACE was not set. Is this running in an Action?" 1>&2
+    exit 1
+fi
+
+cd "$GITHUB_WORKSPACE"
+
+cargo deny check -c /deny.toml $INPUT_CHECK_WHICH
```

Comment:
Where does `$INPUT_CHECK_WHICH` come from?

Reply:
My bad. That is an option to choose which check(s) get performed, defaulting to all. I have added that under `inputs` in `action.yml`.